amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpSinkStage.scala

```diff
@@ -9,22 +9,27 @@ import akka.util.ByteString
 import com.rabbitmq.client.AMQP.BasicProperties
 import com.rabbitmq.client._
 
-final case class OutgoingMessage(bytes: ByteString, immediate: Boolean, mandatory: Boolean, props: Option[BasicProperties])
+final case class OutgoingMessage(bytes: ByteString,
+                                 immediate: Boolean,
+                                 mandatory: Boolean,
+                                 props: Option[BasicProperties])
```

Comment:
BTW, I have learned that I have misliked some particular styling aspece initially but later changed my mind ... maybe the same can happen to you ;-)